ajax/libs/js-xss/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "js-xss",
+  "filename": "xss.min.js",
+  "version": "0.3.3",
+  "description": "Sanitize untrusted HTML (to prevent XSS) with a configuration specified by a Whitelist",
+  "author": "Zongmin Lei <leizongmin@gmail.com> (http://ucdok.com)",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/leizongmin/js-xss.git"
+  },
+  "license": "MIT",
+  "homepage": "http://jsxss.com",
+  "keywords": [
+    "sanitization",
+    "xss",
+    "sanitize",
+    "sanitisation",
+    "input",
+    "security",
+    "escape",
+    "encode",
+    "filter",
+    "validator",
+    "html",
+    "injection",
+    "whitelist"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/leizongmin/js-xss.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "xss*"
```

Comment:
@sajochiu We should use "`**/*.!(html)"` to exclude `test.html` and make it more flexible.